bun/whatsapp-with-vonage/src/main.ts

```diff
@@ -0,0 +1,65 @@
+import jwt from "jsonwebtoken";
+import { fetch } from "undici";
+import { getStaticFiles } from "./utils";
+
+export default async ({ req, res, log, error }) => {
+  if (req.method === "GET") {
+    return res.send(getStaticFiles, 200, {
+      "Content-Type": "text/html; charset=utf-8",
+    });
```

Comment:
Why is it called `getStaticFiles` when it returns the contents of a single file?

Please make a function of format like `getStaticFile(filename: string)` as like https://github.com/appwrite/templates/blob/6dab35f1f221ba075640d93f5a8e374f4fba4349/node/whatsapp-with-vonage/src/main.js#L15